daisy/compute/compute.go

```diff
@@ -169,61 +227,108 @@ func (c *client) CreateInstance(project, zone string, i *compute.Instance) error
 
 // DeleteImage deletes a GCE image.
 func (c *client) DeleteImage(project, name string) error {
-	resp, err := c.raw.Images.Delete(project, name).Do()
+	op, err := c.Retry(c.raw.Images.Delete(project, name).Do)
 	if err != nil {
 		return err
 	}
 
-	return c.i.operationsWait(project, "", resp.Name)
+	return c.i.operationsWait(project, "", op.Name)
 }
 
 // DeleteDisk deletes a GCE persistent disk.
 func (c *client) DeleteDisk(project, zone, name string) error {
-	resp, err := c.raw.Disks.Delete(project, zone, name).Do()
+	op, err := c.Retry(c.raw.Disks.Delete(project, zone, name).Do)
 	if err != nil {
 		return err
 	}
 
-	return c.i.operationsWait(project, zone, resp.Name)
+	return c.i.operationsWait(project, zone, op.Name)
 }
 
 // DeleteInstance deletes a GCE instance.
 func (c *client) DeleteInstance(project, zone, name string) error {
-	resp, err := c.raw.Instances.Delete(project, zone, name).Do()
+	op, err := c.Retry(c.raw.Instances.Delete(project, zone, name).Do)
 	if err != nil {
 		return err
 	}
 
-	return c.i.operationsWait(project, zone, resp.Name)
+	return c.i.operationsWait(project, zone, op.Name)
 }
 
 // GetMachineType gets a GCE MachineType.
 func (c *client) GetMachineType(project, zone, machineType string) (*compute.MachineType, error) {
-	return c.raw.MachineTypes.Get(project, zone, machineType).Do()
+	mt, err := c.raw.MachineTypes.Get(project, zone, machineType).Do()
```

Comment:
Retry takes `func(opts ...googleapi.CallOption) (*compute.Operation, error)` and returns `*compute.Operation`

These functions are all unique in what they return